amazon.sql

```diff
@@ -0,0 +1,77 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='ONLY_FULL_GROUP_BY,STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+-- -----------------------------------------------------
+-- Schema amazon
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema amazon
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `amazon` DEFAULT CHARACTER SET latin1 ;
+USE `amazon` ;
+
+-- -----------------------------------------------------
+-- Table `amazon`.`applicationuser`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `amazon`.`applicationuser` (
```

Comment:
Table name should be a valid name without prefix.... since there will be one user table for whole application.... so  application prefix makes redundant